Fix `NavigateEvent` & `NavigationCurrentEntryChangeEvent` constructor… (#37934)
mdn/content: files/en-us/web/api/navigateevent/navigateevent/index.md

### Parameters

- `type`
  - : A string representing the type of event.
- `init`

>   - : An object that, _in addition of the properties defined in {{domxref("Event/Event", "Event()")}}_, can have the following properties:

Reviewer: ```suggestion
  - : An object that, _in addition to the properties defined in {{domxref("Event/Event", "Event()")}}_, has the following properties:
```

      - : The {{domxref("FormData")}} object representing the submitted data in the case of a `POST` form submission. Defaults to `null`.
    - `hashChange` {{optional_inline}}
      - : A boolean defining if the navigation is a fragment navigation (i.e. to a fragment identifier in the same document). Defaults to `false`.
    - `hasUAVisualTransition` {{optional_inline}}

>       - : A boolean defining if the user agent has performed a visual transition for this navigation before dispatching this event.

Reviewer: ```suggestion
      - : A boolean defining whether the user agent has performed a visual transition for this navigation before dispatching this event. Defaults to `false`.
```
